.github/label-commenter-config.yml

```
@@ -28,7 +28,7 @@ labels:
         action: open
       pr:
         body: "🚨 Alert! Git Police! We couldn’t help but notice that one or more of your commits is missing a sign-off. _A what?_ A commit sign-off (your email address).\n\n
-         To amend the commits in this PR with your signoff using the instructions provided in the DCO check above. \n\n
+         To amend the commits in this PR with your signoff using the instructions provided in the DCO check. \n\n
```

```suggestion
         To amend the commits in this PR with your signoff using the instructions provided in the DCO check above. \n\n
```
Please remove this changes Why did you made this changes